lib/index.ts

```diff
@@ -215,15 +215,20 @@ async function getManifestFiles(targetImage: string, options?: any) {
     return [];
   }
 
-  let excludeGlobs: string[] = [];
+  const excludeGlobs: string[] = ["/sys/**"];
   if (options.manifestExcludeGlobs) {
-    excludeGlobs = options.manifestExcludeGlobs as string[];
+    excludeGlobs.concat(options.manifestExcludeGlobs as string[]);
   }
 
   const globs = options.manifestGlobs as string[];
   const docker = new Docker(targetImage, options);
 
-  let files = await docker.findGlobs(globs, excludeGlobs);
+  let files = await docker.findGlobs(globs, excludeGlobs, "/", false);
+
+  files.forEach(async (dir) => {
+    const nextFiles = await docker.findGlobs(globs, excludeGlobs, dir, true);
+    files.concat(nextFiles);
```

Comment:
this is a NOP: `The concat() method is used to merge two or more arrays. This method does not change the existing arrays, but instead returns a new array.` see: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat

Reply:
thanks! silly me :)